Sortable.js

```diff
@@ -67,7 +67,7 @@
 		$ = win.jQuery || win.Zepto,
 		Polymer = win.Polymer,
 
-		captureMode = {capture: false, passive: false},
+		captureMode = { capture: false, passive: false },
```

Comment:
It is necessary to remove these junk changes, or I can close PR and make your changes myself.